development/architecture/migration-guide/horizontal-migration/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: How to migrate Back Office pages to Symfony with Horizontal migration
+menuTitle: Horizontal migration guide
+chapter: true
+weight: 40
+summary: "The guide we use to migrate pages to Symfony"
+---
+
+# How to migrate Back Office pages to Symfony
```

Comment:
```suggestion
# How to migrate Back Office pages to Symfony using "horizontal" migration approach
```
maybe "horizontal migration" part could lead to other page where we explain base differences compared to vertical migration (when/if we have it)